### PR TITLE
Mobile Wallets Login Flow Fix

### PR DESCRIPTION
**Pull request by alexyoung23j**
## Link to Issue
Closes: #3460 

## Description of Changes
- Fixes the login bugs with mobile login by breaking up login on mobile into two steps, a connect step and a sign step. Each of these is triggered by a button press from the user, preventing the redirect headaches users have experienced. 
- Also resets WC connection on failed login.

## Test Plan
- Tested locally (on mobile via an ngrok url). 

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/modals/login_modal.tsx [hunk 1]
```diff
@@ -397,6 +397,7 @@ export class LoginModal extends ClassComponent<LoginModalAttrs> {
         }}
         setBodyType={(bodyType: LoginBodyType) => {
           this.bodyType = bodyType;
+          redraw();
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/modals/login_modal.tsx:400]: You realize this is a no-op?

**Review by jnaviask — CHANGES_REQUESTED** [review]
Please check in the updated yarn.lock file.

File: packages/commonwealth/client/scripts/views/modals/login_modal.tsx (hunk 1 above)
Comment (alexyoung23j) [reply packages/commonwealth/client/scripts/views/modals/login_modal.tsx:400]: woooops lol

**Review by jnaviask — APPROVED** [review]
Unlikely to cause regressions. We can merge to master for testing on cmn-QA, then ship tomorrow with react.

**Pull request merged**